Projects/RemoteCameraRobot/flask_server.py

```diff
@@ -0,0 +1,77 @@
+from flask import Flask, jsonify, render_template, request, Response, send_from_directory, url_for
+from easygopigo3 import EasyGoPiGo3
+
+MAX_FORCE = 5.0
+MIN_SPEED = 100
+MAX_SPEED = 300
+gopigo3_robot = EasyGoPiGo3()
+
+app = Flask(__name__, static_url_path='')
```

Comment:
Somewhere in here we need a graceful exit that release the port 5000.  If you don't and you exit with CTRL+C a reboot is required because the port remains busy.

Reply:
Alright, this makes sense.